installer/opensuse/defaults

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file is sourced from main.sh to update distro-specific defaults.
+# It must set at least ARCH and MIRROR if not already specified.
+
+if [ -z "$ARCH" ]; then
+    ARCH="$(uname -m)"
+fi
+
+case "$ARCH" in
+arm*) ARCH="armv7hl";;
+arm64 | aarch64) ARCH="arm64";;
```

Comment:
I think this needs to be above the arm* line, otherwise arm64 won't be matched properly

Reply:
good catch!